graphdatascience/tests/integration/test_graph_ops.py

```diff
@@ -672,17 +685,21 @@ def test_graph_relationshipProperties_stream_without_arrow_separate_property_col
 def test_graph_relationships_stream_without_arrow(gds_without_arrow: GraphDataScience) -> None:
     G, _ = gds_without_arrow.graph.project(GRAPH_NAME, "*", ["REL", "REL2"])
 
-    result = gds_without_arrow.beta.graph.relationships.stream(G, ["REL", "REL2"])
+    if gds_without_arrow.server_version() >= ServerVersion(2, 5, 0):
+        result = gds_without_arrow.graph.relationships.stream(G, ["REL", "REL2"])
+    else:
+        result = gds_without_arrow.beta.graph.relationships.stream(G, ["REL", "REL2"])
 
-    expected = gds_without_arrow.run_cypher(
-        "MATCH (n)-[r]->(m) RETURN id(n) AS src_id, id(m) AS trg_id, type(r) AS rel_type"
-    )
+    with pytest.raises(DeprecationWarning):
+        expected = gds_without_arrow.run_cypher(
+            "MATCH (n)-[r]->(m) RETURN id(n) AS src_id, id(m) AS trg_id, type(r) AS rel_type"
+        )
```

Comment:
why is this DeprecationWarning? I thought you filtered the id() warnings?

Reply:
I only filtered it on our endpoints not for every `run_cypher`(YET)

Reply:
I think that's good; let's not filter general queries.

In what endpoints do we explicitly use `id()`?

Reply:
we use it in:

* gds.graph.nodeProperties.stream if people request additional `db_node_properties`
* topological linkPrediction endpoints .. these endpoint work over node objects so we do a MATCH internally
* and gds.find_node_id() .. Used to forward to path algorithms f.i.